packages/fluentui/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -123,13 +120,14 @@ class AccordionTitle extends UIComponent<WithAsProp<AccordionTitleProps>, any> {
           {...accessibility.attributes.content}
           {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.content, unhandledProps)}
         >
-          {Icon.create(indicatorWithDefaults, {
+          {Box.create(indicator === undefined ? {} : indicator, {
```

Comment:
```suggestion
          {Box.create(indicator, {
```

And then:

```
AccordionTitle.defaultProps = {
  indicator: {},
}
```

***

We will have a `propTypes` warning, so:

```diff
-indicator: customPropTypes.shorthand,
+indicator: customPropTypes.shorthandAllowingChildren,
```